packages/core/src/db/models/Dashboard.ts

```
@@ -1,40 +1,158 @@
       if (!dashboard) {
-        throw new Error("Dashboard not found");
+        throw new Error('Dashboard not found');
       }
       return dashboard;
     }
 
+    public static async getDashboards(
+      params: IListParams,
+      user: IUserDocument,
+    ) {
+      const filter = await this.generateFilter(params, user);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)